.ai/auto-review/repository/github.py

```diff
@@ -23,7 +23,24 @@ def post_comment_to_line(self, text, commit_id, file_path, line):
             "body": text,
             "commit_id": commit_id,
             "path" : file_path,
-            "position" : line
+            "side":"RIGHT",
+            "line" : line
+        }
+        response = requests.post(self.__url_add_comment, json = body, headers = headers)
+        if response.status_code == 200 or response.status_code == 201:
+            return response.json()
+        else:
+            raise RepositoryError(f"Error with line comment {response.status_code} : {response.text}")
+        
+    def post_comment_to_line_with_diff_hunks(self, text, commit_id, file_path, line, diff_hunks):
+        headers = self.__header_accept_json | self.__header_authorization
```

Comment:
36 : Incorrect merge of headers, should be one dict, not an or expression of two : Use `headers = {**self.__header_accept_json, **self.__header_authorization}` instead of "|"